pandas_datareader/tiingo.py

```diff
@@ -47,9 +49,13 @@ class TiingoIEXHistoricalReader(_BaseReader):
         session : Session, default None
             requests.sessions.Session instance to be used
         freq : {str, None}
-        Re-sample frequency. Format is # + (min/hour); e.g. "15min" or "4hour".
-        If no value is provided, defaults to 5min. The minimum value is "1min".
-        Units in minutes (min) and hours (hour) are accepted.
+            Re-sample frequency. Format is #min/hour; e.g. "15min" or "4hour".
+            If no value is provided, defaults to 5min. The minimum value is\
+                 "1min".
+            Units in minutes (min) and hours (hour) are accepted.
+        response_format : str, default 'json'
+            Format of response data returned by the underlying Tiingo REST API.
```

Comment:
Is this accurate?   Perhaps something like: "The format to use in the request tot he TIINGO API.  "csv" results in smaller payload, less bandwidth, and may result in smaller charges for accessing the TIINGO API"

Reply:
Sounds good - added wording along those lines. This would affect bandwidth limits and implicitly charges.